tests/backup/inventory_test.js

```diff
@@ -84,7 +84,7 @@ Scenario(
 ).retry(1);
 
 Scenario(
-  'PMM-T961 PMM-T1005 PMM-T1024 Verify create backup modal @backup @bm-mongo',
+  '@PMM-T961 @PMM-T1005 PMM-T1024 Verify create backup modal @backup @bm-mongo',
```

Comment:
What's up with the "@"?

Reply:
it can happen when you have to deliver under pressure, this is acceptable